### DISCUSSION TITLE
Option to configure anki-editior-insert-note to not include *Front and *Back subheadings or Org-capture templates

### DISCUSSION THREAD

**Discussion by nomm-dot** [anki-editor/anki-editor#48] [Ideas]
I might've missed this configuration option, but I couldn't find it anywhere. It would be nice to have another insert-note command that doesn't have the *Front and *Back subheadings, or a way to define it your configuration file as the default. I know other more complex card templates can be achieved with org-capture templates. Another simple solution to just include some useful org-capture templates for anki notes in the repo, I'm super surprised org-capture is not mentioned anywhere actually.

I'm just getting started integrating org-capture into my workflow, so if I create some good templates that work for me I'll just put them below, but I really only use the Basic template at the moment since it was faster than switching between note types when creating cards in the Anki gui.

**Comment by slotThe** [comment]
I think that the inserted note should correspond to how Anki treats notes rather closely, so inserting subheadings looks, to me, to be the right thing to do.

Capture templates would be most appreciated, however! We can probably find a place in the README to put these
